skills/lark-approval/references/lark-approval-tasks-query.md

```
@@ -0,0 +1,89 @@
+# 表格格式输出，便于快速浏览
+lark-cli approval tasks query --params '{"topic":"1"}' --format table --as user
+```
+
+## 参数
+
+| 参数 | 必填 | 说明 |
+|------|------|------|
+| `--params '{"topic":"..."}'` | 是 | 查询参数，使用 JSON 传入 |
+| `topic` | 是 | 任务分组主题，见下方“topic 枚举” |
+| `definition_code` | 否 | 审批定义 Code，用于仅查询某个审批定义下的任务 |
+| `locale` | 否 | 返回语言：`zh-CN` \\| `en-US` \\| `ja-JP` |
```

markdownlint-cli2 (0.22.1)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)